### PR TITLE
Added flight timetable API and exercises data API

### PR DESCRIPTION
**Pull request by manishkumar-hub**

### CODE REVIEW

**Comment by manishkumar-hub** [comment]
I have added two working APIs . Please review it .

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Please rename the markdown file names to be lower case. For e.g. `flight-timetable.md`
`exercise.jpg`

File: src/content/apis/health/exercises.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Exercises API 
+description:  Access thousands of exercises with individual exercise data and animated demonstrations 
+categories:
+  - health
+features:
+  https: true
+url: 'https://rapidapi.com/ExoWatts/api/exercises2/'
+logo: exercise.jpg
```
[the remark this answers is not in the document]
Comment (manishkumar-hub) [reply src/content/apis/health/exercises.md]: Updated

**Comment by TheLearneer** [comment]
@manishkumar-hub please make sure that all the files you added are completely lower case 🙏

**Comment by TheLearneer** [comment]
@manishkumar-hub Prettier is actually failing.
Can you please run `npm run format` or `yarn format` on the project directly and then update the changes 🙇

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Tests seems to be failing, please run `npm run format` or `yarn format` and push the changes 🙏

**Comment by TheLearneer** [comment]
Hi @manishkumar-hub the tests are failing.
Now the only thing you need to do is run `npm run format` or `yarn format` in your terminal where you have cloned the project.

No other changes are necessary 🙇

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Please remove the changes made to lock files 😓

**Comment by manishkumar-hub** [comment]
Sorry for the incoveniance caused . I have undo the changes of lock file.  I hope it is fine now .

**Review by TheLearneer — APPROVED** [review]
Finally 😌 
I hope you learnt a thing or two @manishkumar-hub 😉 

Thank you for your contribution 🙏

File: src/content/apis/health/exercises.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Exercises-API
```
Comment (TheLearneer) [inline src/content/apis/health/exercises.md:2]:
```suggestion
name: Exercises API
```

File: src/content/apis/travel/flight-timetable.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+name: TimeTable-Lookup
```
Comment (TheLearneer) [inline src/content/apis/travel/flight-timetable.md:2]:
```suggestion
name: TimeTable Lookup
```

**Review by TheLearneer — CHANGES_REQUESTED** [review]

**Review by TheLearneer — APPROVED** [review]
🙏

**Comment by manishkumar-hub** [comment]
Thanks a lot !! ☺️

**Pull request merged**